utils/latency.py

```diff
@@ -1,56 +1,40 @@
 from datetime import datetime
-from datetime import timedelta
 import numpy as np
 import sys
 
-if len(sys.argv) < 4:
-    print(
-        f'Usage {sys.argv[0]} "$(date +%z)" '
-        f'ovn-binding.log ovn-installed.log'
-    )
+if len(sys.argv) < 3:
+    print(f'Usage {sys.argv[0]} ' f'ovn-binding.log ovn-installed.log')
 
-tz = -int(sys.argv[1])
-delta = timedelta(hours=int(tz / 100), minutes=int(tz % 100))
```

Comment:
Doesn't the time zone not matter so long as all containers are running the same time zone? The ovn-tester container uses the same base image as the other containers in the test, so it should be using the same time zone as the others. So whether they all use UTC or whether they all use UTC+3 or UTC-4, it shouldn't really matter.

Reply:
The problem is that OVS daemons (ovn-controller) logs always in UTC, regardless of the time zone of the system/container.  You can see the `Z` a.k.a 'Zulu time' in the ovn-controller log.  So, if the time zone configured in the container is not UTC, results will be skewed.

Reply:
Ah, OK, and since the ovn-installed times are pulled from ovn-controller logs, then that could potentially cause an issue. I'll add the timedelta calculation back in.

Reply:
There seems to be a way to subclass the logging formatter so ovn-tester will log in UTC.  Like this:
```
class UTCFormatter(logging.Formatter):
    converter = time.gmtime
```
Taken from here: https://docs.python.org/3/howto/logging-cookbook.html#formatting-times-using-utc-gmt-via-configuration